jck/jtrunner/JavatestUtil.java

```diff
@@ -839,9 +845,10 @@ private static boolean generateJTB() throws Exception {
 			// Get any additional jvm options for specific tests.
 			extraJvmOptions += getTestSpecificJvmOptions(jckVersion, tests);
 			extraJvmOptions += suppressOutOfMemoryDumpOptions;
+			
 
 			// Add the JVM options supplied by the user plus those added in this method to the jtb file option.
-			fileContent += "set jck.env.devtools.refExecute.otherOpts \" " + extraJvmOptions + " \"" + ";\n";	
+			fileContent += "set jck.env.devtools.refExecute.otherOpts \" " + extraJvmOptions + " " + jvmOpts + " \"" + ";\n";	
```

Comment:
https://github.com/adoptium/aqa-tests/blob/6c6b9a5faaee7d1f8a2a3fb861fde3a1c554533b/jck/jtrunner/JavatestUtil.java#L678 should be updated as well.